db/seeds.rb

```diff
@@ -10,3 +10,4 @@
 import_fixture 'practices'
 import_fixture 'categories'
 import_fixture 'learnings'
+import_fixture 'reports'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.